### PR TITLE
Add Google authentication

### PR DESCRIPTION
**Pull request by DonnyLp**

### CODE REVIEW

**Review by DerrickMac — APPROVED** [review]
This looks good to me!

File: backend/config/googleAuthConfig.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+            //log the user in
+            done(null,user);
+        }
+        else{
+            const newUser = new User({
+            username: profile.displayName,
+            email: profile.emails[0].value,
+            fName: profile.name.givenName,
+            lName: profile.name.familyName,
+            profilePicture: '',
```
Comment (ItsJonasL) [inline backend/config/googleAuthConfig.js]: should this not be profile.photos[0].value?

**Review by ItsJonasL — CHANGES_REQUESTED** [review]

File: backend/config/googleAuthConfig.js (hunk 1 above)
Comment (DonnyLp) [reply backend/config/googleAuthConfig.js]: Yes, you're right! Should I fix this now?

Comment (ItsJonasL) [reply backend/config/googleAuthConfig.js]: Yes. I can approve it and push right away.

**Review by ItsJonasL — APPROVED** [review]
Approved. Thanks for the fast turn around

**Pull request merged**